src/ios/ConnectSDKCordovaDispatcher.m

```diff
@@ -966,26 +967,43 @@ - (void) displayMediaCommon:(JSCommand*)command type:(NSString*)type
     NSURL* url = [NSURL URLWithString:command.args[@"url"]];
     NSString* mimeType = command.args[@"mimeType"];
     NSDictionary* options = command.args[@"options"];
-    
-    NSString* title = nil;
-    NSString* description = nil;
-    NSURL* iconUrl = nil;
+
+    MediaInfo *mediaInfo = [[MediaInfo alloc] initWithURL:url mimeType:mimeType];
     BOOL shouldLoop = NO;
```

Comment:
@eunikolsky @frolov-alexey  To have IOS and Android consistency please declare title,description and icon url and subtitle variables and assign values to them Or do vice versa. Your call